src/src/components/home.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+
+
+class Home extends Component {
```

Comment:
you should practice using a descriptive names to your components. For example Home is a very generic name, you should give it name like 'List of users' since the component is doing just that, listing all users.